echo-test/src/main/groovy/com/netflix/spinnaker/echo/test/RetrofitStubs.groovy

```diff
@@ -64,7 +64,7 @@ trait RetrofitStubs {
   }
 
   BuildEvent createBuildEventWith(BuildEvent.Result result) {
-    def build = result ? new BuildEvent.Build(result == BUILDING, 1, result, null, []) : null
+    def build = result ? new BuildEvent.Build(result == BUILDING, '1', result, null, []) : null
```

Comment:
SUPER minor, but what about adding a test for a non-numeric build number as well as a numeric build number? :P And maybe a call out/comment on why?  I like this overall though!